### ISSUE TITLE
VB Expressions fail with netcore3

### ISSUE THREAD

**Issue by dmetzgar** [UiPath/CoreWF#86]
A test that use VB expressions in a XAML that is loaded with ActivityXamlServices.Load fails. The test is [here](https://github.com/UiPath/corewf/blob/b38ca67bfb0ae1206bbfe65deb629a3fbfa63a0d/src/Test/TestCases.Workflows/WF4Samples/Expressions.cs#L28).

The failure is "error BC35000: Requested operation is not available because the runtime library function 'Microsoft.VisualBasic.CompilerServices.ProjectData.SetProjectError' is not defined."

Full stack is below:
```
    System.Activities.InvalidWorkflowException : The following errors were encountered while processing the workflow tree:
    'DynamicActivity': The private implementation of activity '1: DynamicActivity' has the following validation error:   Compiler error(s) encountered processing expression "New SalaryStats()".Microsoft.CodeAnalysis.Scripting.CompilationErrorException: (1) : error BC35000: Requested operation is not available because the runtime library function 'Microsoft.VisualBasic.CompilerServices.ProjectData.SetProjectError' is not defined.
       at Microsoft.CodeAnalysis.Scripting.ScriptBuilder.ThrowIfAnyCompilationErrors(DiagnosticBag diagnostics, DiagnosticFormatter formatter)
       at Microsoft.CodeAnalysis.Scripting.ScriptBuilder.CreateExecutor[T](ScriptCompiler compiler, Compilation compilation, Boolean emitDebugInformation, CancellationToken cancellationToken)
       at Microsoft.CodeAnalysis.Scripting.Script`1.GetExecutor(CancellationToken cancellationToken)
       at Microsoft.CodeAnalysis.Scripting.Script`1.CommonGetExecutor(CancellationToken cancellationToken)
       at Microsoft.CodeAnalysis.Scripting.Script`1.TryGetPrecedingExecutors(Script lastExecutedScriptInChainOpt, CancellationToken cancellationToken)
       at Microsoft.CodeAnalysis.Scripting.Script`1.GetPrecedingExecutors(CancellationToken cancellationToken)
       at Microsoft.CodeAnalysis.Scripting.Script`1.RunAsync(Object globals, Func`2 catchException, CancellationToken cancellationToken)
       at Microsoft.VisualBasic.Activities.HostedCompiler.CompileExpression(String expressionString, Func`2 getVariableType, ScriptOptions options, Type lambdaReturnType) in C:\dev\uipath\corewfproper\src\CoreWf\Microsoft\VisualBasic\Activities\HostedCompiler.cs:line 37
       at Microsoft.VisualBasic.Activities.VisualBasicHelper.Compile[T](LocationReferenceEnvironment environment, Boolean isLocationReference) in C:\dev\uipath\corewfproper\src\CoreWf\Microsoft\VisualBasic\Activities\VisualBasicHelper.cs:line 507
    'DynamicActivity': The private implementation of activity '1: DynamicActivity' has the following validation error:   Compiler error(s) encountered processing expression "New Employee("Frank", "Kimono", 89000.0)".Microsoft.CodeAnalysis.Scripting.CompilationErrorException: (1) : error BC35000: Requested operation is not available because the runtime library function 'Microsoft.VisualBasic.CompilerServices.ProjectData.SetProjectError' is not defined.
       at Microsoft.CodeAnalysis.Scripting.ScriptBuilder.ThrowIfAnyCompilationErrors(DiagnosticBag diagnostics, DiagnosticFormatter formatter)
       at Microsoft.CodeAnalysis.Scripting.ScriptBuilder.CreateExecutor[T](ScriptCompiler compiler, Compilation compilation, Boolean emitDebugInformation, CancellationToken cancellationToken)
       at Microsoft.CodeAnalysis.Scripting.Script`1.GetExecutor(CancellationToken cancellationToken)
       at Microsoft.CodeAnalysis.Scripting.Script`1.CommonGetExecutor(CancellationToken cancellationToken)
       at Microsoft.CodeAnalysis.Scripting.Script`1.TryGetPrecedingExecutors(Script lastExecutedScriptInChainOpt, CancellationToken cancellationToken)
       at Microsoft.CodeAnalysis.Scripting.Script`1.GetPrecedingExecutors(CancellationToken cancellationToken)
       at Microsoft.CodeAnalysis.Scripting.Script`1.RunAsync(Object globals, Func`2 catchException, CancellationToken cancellationToken)
       at Microsoft.VisualBasic.Activities.HostedCompiler.CompileExpression(String expressionString, Func`2 getVariableType, ScriptOptions options, Type lambdaReturnType) in C:\dev\uipath\corewfproper\src\CoreWf\Microsoft\VisualBasic\Activities\HostedCompiler.cs:line 37
       at Microsoft.VisualBasic.Activities.VisualBasicHelper.Compile[T](LocationReferenceEnvironment environment, Boolean isLocationReference) in C:\dev\uipath\corewfproper\src\CoreWf\Microsoft\VisualBasic\Activities\VisualBasicHelper.cs:line 507
    'DynamicActivity': The private implementation of activity '1: DynamicActivity' has the following validation error:   Compiler error(s) encountered proce<truncated> The exception message has been truncated due to size.  Use ActivityValidationServices to see all violations.
```

**Comment by dmetzgar** [comment]
Based on the error message and [this description](https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/error-messages/bc35000) of the error code, it seems that the compiler code is assuming that the Microsoft.VisualBasic.dll is included in the references. It could be that Roslyn has a different code path for netcore3.

**Comment by lbargaoanu** [comment]
`ProjectData` is forwarded from Microsoft.VB to Microsoft.VB.Core. And there is a reference to Microsoft.VB.

**Comment by lbargaoanu** [comment]
OK, so it was using the VB runtime version used by Roslyn. I've added a reference to the newest version. Fixed in #87.
